src/desktop/components/split_test/splitTestMiddleware.ts

```diff
@@ -15,9 +14,7 @@ export function splitTestMiddleware(
     const name = key.toUpperCase()
     if (!res.locals.sd[name]) {
       const test = new SplitTest(req, res, configuration)
-      const outcome = test.outcome()
-
-      updateSharifyAndContext(res, name, outcome)
```

Comment:
@mzikherman - will reverting this line impact any of the other infra that was put in place around the refactored express-context work that was done last week?

Reply:
No, it just means that AB tests wouldn't 'work' in that, the value is no longer getting set in the context, for code to introspect on. But, there's also no currently running AB tests.

I suppose the PR which launches the test (which this is a revert of), wound up doing several things:
  - remove unused page cache middleware
  - update AB test setup to use new context
  - specific changes to implement/launch the AB test
 
scope creep! In hindsight, those could have all been smaller PR's, and then the ultimate revert to close the test would have been a revert of that last PR in the sequence.